pallets/runtime/tests/src/ext_builder.rs

```diff
@@ -97,6 +98,8 @@ pub struct ExtBuilder {
     adjust: Option<Box<dyn FnOnce(&mut Storage)>>,
```

Comment:
Btw, if you don't expect there to be reuse of `.bridge_txs(...)` then `.adjust(...)` is imo a fine/better way to handle one-offs like this.